index.html

```diff
@@ -14,145 +14,168 @@
+            <option value="slideInLeft">slideInLeft</option>
+            <option value="slideInRight">slideInRight</option>
+          </optgroup>
+
+          <optgroup label="Sliding Exits">
+            <option value="slideOutUp">slideOutUp</option>
+            <option value="slideOutDown">slideOutDown</option>
+            <option value="slideOutLeft">slideOutLeft</option>
+            <option value="slideOutRight">slideOutRight</option>
+          </optgroup>
+
+          <optgroup label="Zoom Entrances">
+            <option value="zoomIn">zoomIn</option>
+            <option value="zoomInDown">zoomInDown</option>
+            <option value="zoomInLeft">zoomInLeft</option>
+            <option value="zoomInRight">zoomInRight</option>
+            <option value="zoomInUp">zoomInUp</option>
+          </optgroup>
+
+          <optgroup label="Zoom Exits">
+            <option value="zoomOut">zoomOut</option>
+            <option value="zoomOutDown">zoomOutDown</option>
+            <option value="zoomOutLeft">zoomOutLeft</option>
+            <option value="zoomOutRight">zoomOutRight</option>
+            <option value="zoomOutUp">zoomOutUp</option>
+          </optgroup>
+
+          <optgroup label="Specials">
+            <option value="hinge">hinge</option>
+            <option value="rollIn">rollIn</option>
+            <option value="rollOut">rollOut</option>
+          </optgroup>
+        </select>
+
+        <button class="butt js--triggerAnimation">Animate it</button>
+      </form>
+      <hr />
+      <p class="meta"><a class="js--triggerSection" href="javascript:void(0);">Builder</a> or <a href="https://raw.github.com/daneden/animate.css/master/animate.css" download="animate.css">Download Animate.css</a> or <a href="//github.com/daneden/animate.css">View on GitHub</a></p>
+      <p><small>Another thing from <a href="//daneden.me">Daniel Eden</a>.</small></p>
+    </div>
+  </main><!-- /.site__content -->
+</section><!-- /Section: Animations Exbihition -->
+
+
+<section class="site__section section__builder island" style="display:none;"><!-- Section: Builder -->
+
+  <header class="site__header">
+    <div class="wrap">
+     <span id="animationSandbox" style="display: block;"><h1 class="site__title mega">Animate.css Builder</h1></span>
+      <a href="javascript:void(0);" class="js--triggerSection">Voltar</a>
+    </div>
+  </header><!-- /.site__header -->
+
+  <main class="site__content island" role="content">
+    <div class="wrap">
+      <form class="js--builder">
+        <button class="butt">Build</button>
+      </form>
+      <hr />
+      <p class="meta"><a class="js--triggerSection" href="javascript:void(0);">Animate.css</a> or <a href="https://raw.github.com/daneden/animate.css/master/animate.css" download="animate.css">Download Animate.css</a> or <a href="//github.com/daneden/animate.css">View on GitHub</a></p>
+      <p><small>Developed by Igorxp5</small></p>
```

Comment:
Rather than adding credits for individuals, let's remove the credit here (and the one to myself on line 151) and provide a link to the project's [contributors](https://github.com/daneden/animate.css/graphs/contributors)